app/controllers/users_controller.rb

```diff
@@ -23,4 +24,16 @@ def user_params
     params.require(:user).permit(:name, :role)
   end
   
+  def get_collaborated_wikis
```

Comment:
You could do this with ActiveRecord, which would be more performant. eg. `@collaborated_wikis = current_user.collaborations`.